html-relative-links/about-me/about-me.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>About Me</title>
+  <style>
+    img {
+      width: 300px;
+    }
+  </style>
+</head>
+<body>
+  <ul>
+    <li><a href="/html-relative-links/index.html">Home Page</a></li>
+    <li><a href="/html-relative-links/about-me/about-me.html">About Me</a></li>
+    <li><a href="/html-relative-links/projects/projects.html">Projects</a></li>
+    <li><a href="/html-relative-links/contact-us/contact-us.html">Contact Us</a></li>
```

Comment:
You can access different directories by using this type of notation - `../` to traverse to a parent directory, or `../../` for a grandparent directory and so on.

Switch to this notation instead of `/html-relative-links/`